### ISSUE TITLE
Deploying several azurerm_function_app

### ISSUE THREAD

**Issue by JuanFcoMiranda** [Azure-Samples/azure-functions-event-grid-terraform#3]
Hello, it's really helpfull this repo, but I have a doubt, could we use it to deploy several azurerm_function_app at the same time??

Kind regards.

**Comment by hannesne** [comment]
Yes you can use this approach - I would put multiple references of the function module in the main.tf, passing different values to each, for each app you would like to deploy.

**Comment by JuanFcoMiranda** [comment]
Thank you very much, and another question, could I assign a event suscription and a function to every topic?

**Comment by hannesne** [comment]
Yes, again, you would replicate the part that does the assignment for each topic\function combination, as long as you don't exceed the limits imposed by event grid, as described here: https://docs.microsoft.com/en-us/azure/event-grid/quotas-limits
